packages/documentation-site/patternfly-docs/content/PatternFly-AI/conversation-design/conversation-design.md

```diff
@@ -0,0 +1,75 @@
+---
+id: Conversation design
+section: PatternFly-AI
```

Comment:
Which sections do we want this and the chatbots.md files to appear in the nav? Should they be under patternfly-ai/conversation-design as different tabs?

if so, i think you should be able to do that by using 
```
id: (page title)
section: PatternFly-AI
source: (page title again? ... really it's whatever you want to be on the tab)
```